src/main/server/api/v1/interfaces/messageInterface.ts

```diff
@@ -37,6 +37,7 @@ export class MessageInterface {
     static async sendMessageSync(
         chatGuid: string,
         message: string,
+        attributedBody: Record<string, any>,
```

Comment:
This needs to be optional. This interface handler is for both the private API and AppleScript. So we need to make it optional since it's not something that the AppleScript can implement

Reply:
That was a typing error, sendMessageSync still works on a null value, but should be resolved now